src/components/PostList/PostList.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { Post } from '../Post';
+
+import './PostList.scss';
+
+export const PostList = ({ posts }) => (
+  <div className="list">
+    {posts.map(post => (
+      <Post {...post} key={post.id} />
```

Comment:
Краще явно передавати пропси
```suggestion
      <Post post={post} key={post.id} />
```